recipes/newrelic/infrastructure/logs/debian-logs.yml

```diff
@@ -107,9 +107,8 @@ install:
           sudo touch /etc/newrelic-infra/logging.d/logging.yml;
         - |
           # Enable infra-agent logs to New Relic
-          sudo sed -i "s,.*verbose:.*,," /etc/newrelic-infra.yml
           sudo sed -i "s,.*log_format:.*,," /etc/newrelic-infra.yml
-          echo -e "verbose: 3\nlog_format: json" | sudo tee -a /etc/newrelic-infra.yml > /dev/null
+          echo -e "log_format: json" | sudo tee -a /etc/newrelic-infra.yml > /dev/null
           sudo sed -i '/^$/d' /etc/newrelic-infra.yml
```

Comment:
I think we can just remove this whole block

Reply:
Yeah @jbeveland27, Jose confirmed we can remove verbose. Why did we put log_format json (if you remember)?

Reply:
It's the recommended format when forwarding the agent's logs: https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent/#agent-logs

Reply:
So I think it's all the same body of work that we can remove

Reply:
Checking what's the default value in the agent, if it's json I'll remove

Reply:
Ok the default log format is "text", so I think we need to keep our override to json?

Reply:
I'm not sure keeping it will have any affect, so it could probably go either way.

Going back to the first commit for the logs recipe, both `verbose: 3` and `log_format: json` were present: https://github.com/newrelic/open-install-library/blob/21e9dd1009a20e940827a49f68aa5f45dcae0763/recipes/newrelic/logs/logs.yml#L53

The [docs](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#log-format) indicate `log_format` just defines the log output format. Since we're going back to inheriting the infra-agent default, I don't see a compelling reason to keep `log_format: json`.

Reply:
Jose confirmed we can remove and rely on the default (text).